pkgs/top-level/python-packages.nix

```diff
@@ -14126,7 +14122,6 @@ self: super: with self; {
   };
 
   torchWithCuda = self.torch.override {
-    magma = pkgs.magma-cuda-static;
```

Comment:
I'm flip-flopping between being okay with this and the rocm variant gone from top level and not.
I am somewhat partial to doing a ``gpuBackend`` solution and keeping this and the equivalent on ``torchWithRocm`` here.

Reply:
Marking resolved as per https://github.com/NixOS/nixpkgs/pull/268746#discussion_r1400351241